build_tools/azure/posix-32.yml

```diff
@@ -0,0 +1,55 @@
+parameters:
+  name: ''
+  vmImage: ''
+  matrix: []
+
+jobs:
+- job: ${{ parameters.name }}
+  pool:
+    vmImage: ${{ parameters.vmImage }}
+  variables:
+    TEST_DIR: '$(Agent.WorkFolder)/tmp_folder'
+    JUNITXML: 'test-data.xml'
+    OMP_NUM_THREADS: '4'
+    OPENBLAS_NUM_THREADS: '4'
+    SKLEARN_SKIP_NETWORK_TESTS: '1'
+  strategy:
+    matrix:
+      ${{ insert }}: ${{ parameters.matrix }}
+
+  steps:
+    - script: >
+        docker container run --rm
+        -v $TEST_DIR:/temp_dir
+        -v $PWD:/io
+        -w /io
+        -d
+        --name skcontainer
+        -e DISTRIB=ubuntu-32
+        -e TEST_DIR=/temp_dir
+        -e JUNITXML=$JUNITXML
+        -e VIRTUALENV=testvenv
+        -e JOBLIB_VERSION=$JOBLIB_VERSION
+        -e SKLEARN_NO_OPENMP=$SKLEARN_NO_OPENMP
+        -e OMP_NUM_THREADS=$OMP_NUM_THREADS
+        -e OPENBLAS_NUM_THREADS=$OPENBLAS_NUM_THREADS
+        -e SKLEARN_SKIP_NETWORK_TESTS=$SKLEARN_SKIP_NETWORK_TESTS
+        i386/ubuntu:16.04
+        sleep 1000000
```

Comment:
I've not seen this trick before!

Reply:
Actually I do not understand: why are the subsequent steps being executed before the completion of this step?

Edit:

Oh I seed, you used the `-d` flag to detach the container and let it run in the background. Maybe you could explain this with a comment.

Reply:
Added a comment about using detached along with using `-detach` to be more explicit.

Reply:
If the installed version of sleep supports units other than seconds, please specify in minutes.